meta/run.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+print_help() {
+    NAME=$(basename "$ARG0")
+    cat <<EOF
+Usage: $NAME [arguments...] [SOURCEFILE]
```

Comment:
Mind you that jakt might gain other arguments in the future. Just replace "SOURCEFILE" with "arguments to jakt" or something. Also see #213 .

Reply:
Fixed.